content/en/blog/_posts/2022-02-09-sig-node-ci-subproject-celebrates/index.md

```diff
@@ -0,0 +1,186 @@
+fixed and the job was passing without flakes.
+
+[![Slack announcement that Serial tests are green](serial-tests-green.png)](https://kubernetes.slack.com/archives/C0BP8PW9G/p1638211041322900)
+
+The 1.23 release got a special shout out for the tests quality and CI signal.
+The SIG Node CI subproject was proud to have helped contribute to such
+a high quality release, in part due to our efforts in identifying
+and fixing flakes in Node and beyond.
+
+[![Slack shoutout that release was mostly green](release-mostly-green.png)](https://kubernetes.slack.com/archives/C92G08FGD/p1637175755023200)
+
+***December 2021.*** An estimated 90% of test jobs were passing at the time of
+the 1.23 release (up from 42% in August 2020).
+
+Dockershim was removed from Kubernetes. This affected nearly half of SIG Node's
+test jobs and the SIG Node CI subproject reacted quickly and retargeted all the
+tests. SIG Node was the first SIG to complete test migrations off dockershim,
+providing examples for other affected SIGs. The vast majority of new jobs passed
+at the time of introduction without further fixes required. There are still
+some wrinkles from the dockershim removal as we uncover more dependencies on
+dockershim, but we plan to stabilize all test jobs by the 1.24 release.
+
+## Statistics
+
+Our regular meeting attendees and subproject participants for the few past month:
+
+- Aditi Sharma
+- Artyom Lukianov
+- Arnaud Meukam
+- Danielle Lancashire
+- David Porter
+- Davanum Srinivas
+- Elana Hashman
+- Francesco Romani
+- Matthias Bertschy
+- Mike Miranda
+- Paco Xu
+- Peter Hunt
+- Ruiwen Zhao
+- Ryan Phillips
+- Sergey Kanzhelev
+- Skyler Clark
+- Swati Sehgal
+- Wenjun Wu
+
+The kubernetes/test-infra repository contains test definitions.The number of
+Node PRs just in that repository:
+- 2020 PRs (since May): [183](https://github.com/kubernetes/test-infra/pulls?q=is%3Apr+is%3Aclosed+label%3Asig%2Fnode+created%3A2020-05-01..2020-12-31+-author%3Ak8s-infra-ci-robot+)
+- 2021 PRs: [264](https://github.com/kubernetes/test-infra/pulls?q=is%3Apr+is%3Aclosed+label%3Asig%2Fnode+created%3A2021-01-01..2021-12-31+-author%3Ak8s-infra-ci-robot+)
+
+Triaged issues and PRs on CI board (including triaging away from the subgroup scope):
+
+- 2020 (since May): [132](https://github.com/issues?q=project%3Akubernetes%2F43+created%3A2020-05-01..2020-12-31)
+- 2021: [532](https://github.com/issues?q=project%3Akubernetes%2F43+created%3A2021-01-01..2021-12-31+)
+
+## Future
+
+Just keeping the "lights on" is a bold task and we are committed to get going.
+We are working to simplify the triage and reviews experience.
+
+Specifically, we are working on better tests organization, naming,
+and improved test tracking:
+
+- https://github.com/kubernetes/enhancements/pull/3042
+- https://github.com/kubernetes/test-infra/issues/24641
+- [Kubernetes SIG-Node CI Testgrid Tracker](https://docs.google.com/spreadsheets/d/1IwONkeXSc2SG_EQMYGRSkfiSWNk8yWLpVhPm-LOTbGM/edit#gid=0)
```

Comment:
We should add a mention of who is authorized to view this sheet (or omit it, if it's only a small audience).

Reply:
Fixed permissions. everyone can view it now